source/extensions/tracers/zipkin/zipkin_tracer_impl.h

```diff
@@ -85,6 +85,9 @@ class ZipkinSpan : public Tracing::Span {
 
   std::string getTraceIdAsHex() const override { return span_.traceIdAsHexString(); };
 
+  // TODO: This method is unimplemented for Zipkin.
```

Comment:
nit: a link to an issue with the follow-up work would be better (here and other stubs).

Reply:
I covered this in https://github.com/envoyproxy/envoy/pull/33909/commits/7344695628ace7828f6ef4220f97fc1f4c4f8c50 - let me know if that looks better